protoc

```diff
@@ -14,10 +14,13 @@ protocversion=3.0.0
 protocurl="https://github.com/google/protobuf/releases/download/v${protocversion}/protoc-${protocversion}-${os}-${arch}.zip"
 
 if [ ! -f "$protocbin" ]; then
-  echo "downloading $protocbin" >&2
+  echo "downloading $protocbin"
```

Comment:
I thought there was some reason why I deliberately avoided logging to stdout here... But I now can't imagine why that would be a problem.  So this is probably fine.